exercises/say/say_test.exs

```diff
@@ -0,0 +1,90 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("say.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule SayTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "says zero in english" do
+    assert Say.in_english(0) == "zero"
+  end
+
+  @tag :pending
+  test "says one in english" do
+    assert Say.in_english(1) == "one"
+  end
+
+  @tag :pending
+  test "says fourteen in english" do
+    assert Say.in_english(14) == "fourteen"
+  end
+
+  @tag :pending
+  test "says twenty in english" do
+    assert Say.in_english(20) == "twenty"
+  end
+
+  @tag :pending
+  test "says twenty two in english" do
+    assert Say.in_english(22) == "twenty-two"
+  end
+
+  @tag :pending
+  test "says one hundred in english" do
+    assert Say.in_english(100) == "one hundred"
+  end
+
+  @tag :pending
+  test "says one hundred twenty three in english" do
+    assert Say.in_english(123) == "one hundred twenty-three"
+  end
+
+  @tag :pending
+  test "says one thousand in english" do
+    assert Say.in_english(1_000) == "one thousand"
+  end
+
+  @tag :pending
+  test "says one thousand two hundred thirty four in english" do
+    assert Say.in_english(1_234) == "one thousand two hundred thirty-four"
+  end
+
+  @tag :pending
+  test "says one million in english" do
+    assert Say.in_english(1_000_000) == "one million"
+  end
+
+  @tag :pending
+  test "says one million two thousand three hundred forty-five in english" do
+    assert Say.in_english(1_002_345) == "one million two thousand three hundred forty-five"
+  end
+
+  @tag :pending
+  test "says one billion in english" do
+    assert Say.in_english(1_000_000_000) == "one billion"
+  end
+
+  @tag :pending
+  test "says a big number in english" do
+    assert Say.in_english(987_654_321_123) == "nine hundred eighty-seven billion six hundred fifty-four million three hundred twenty-one thousand one hundred twenty-three"
+  end
+
+  @tag :pending
+  test "numbers below zero are out of range" do
+    assert_raise ArgumentError, fn ->
```

Comment:
@zporter we have been embracing the use of `{:ok, result}` or `{:error, "message"}` idiom in most of our newer exercises. Can you please update this exercise to that style?